corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -709,7 +709,6 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
 
     const FormMenuView = Marionette.CollectionView.extend({
         template: _.template($("#form-menu-template").html() || ""),
-        tagName: 'li',
```

Comment:
why does this need to be removed?

Reply:
The dropdown was getting rendered under a <li> tag and that caused a bullet icon to appear in front of the dropdown. Removing this line makes the dropdown render under a <div> tag.